actions/handle_caches/action.yaml

```diff
@@ -21,3 +24,12 @@ runs:
         libint-2.6.0/**/
         install/**/
       key: libint-gcc${{ matrix.gcc_version }}
+  - if: ${{contains(inputs.dependencies, 'libfort')}}
+    name: cache-libfort  
+    id: cacheLibfort
+    uses: actions/cache@v2
+    with:
+      path: |
+        libfort-0.4.2/**/
```

Comment:
We really need to avoid hard-coding versions like this. If versions change it's annoying to hunt down all the spots where we have the version.

Reply:
Yes. I guess we can avoid hard coding. Let me try.